benedict/serializers/__init__.py

```diff
@@ -37,6 +39,7 @@
     'yaml': _YAML_SERIALIZER,
     'yml': _YAML_SERIALIZER,
     'xml': _XML_SERIALIZER,
+    'ini': _INI_SERIALIZER,
```

Comment:
These entries are sorted alphabetically.

Reply:
Oh alright, this can be fixed